### ISSUE TITLE
Inclusion of ORM into Iris?

### ISSUE THREAD

**Issue by bigcurl** [kataras/iris#629]
Would be great to see the either inclusion of an existing orm or the creation of a new orm in the sense of ActiveRecord from Rails or ecto from Elixir.

**Comment by kataras** [comment]
Exactly, Iris doesn't provides an ORM for these things, gives you the freedom to choice without limit you with specific options to select.

This was also questioned before and all users suggested the `gorm` (I'm using this too)
